src/sidebar/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="utf-8" />
+    <title>Firefox Notes</title>
+    <link href="static/css/menu.css" rel="stylesheet" />
+    <link href="static/css/styles.css" rel="stylesheet" />
+  </head>
+  <body>
+    <div id="notes"></div>
+  </body>
+  <script src="vendor/ckeditor.js"></script>
+  <script defer src="vendor/material.js"></script>
+  <script src="app.js"></script>
```

Comment:
Nit: the `<script>` tags should probably be within the `<body>` tag, instead of orphaned here.

Reply:
Until yesteday, we were powering react directly on body, which clear its content and explain why script were after body. That's bad practice and has been changed now using a div. I will move those back inside body, thanks.